src/NUnitFramework/testdata/AssertMultipleData.cs

```diff
@@ -260,6 +265,88 @@ public void AssumptionInBlock()
                 Assume.That(2 + 2 == 4);
             });
         }
+
+#if ASYNC
+        [Test]
+        public void ThreeAssertsSucceed_Async()
+        {
+            Assert.Multiple(async () =>
+            {
+                await Task.Delay(1000);
```

Comment:
Does this delay (and the 500 below) have to be so long? Our tests are already getting fairly long to run...

Would something like `await Task.FromResult(false)` suffice?